coffee-grpc/coffee-grpc-server-extension/src/main/java/hu/icellmobilsoft/coffee/grpc/server/GrpcServerManager.java

```diff
@@ -80,7 +80,7 @@ public class GrpcServerManager {
     @GrpcServerConnection(configKey = IGrpcServerConfig.DEFAULT_SERVER_KEY)
     private GrpcServerConfig serverConfig;
 
-    @Resource
+    @Resource(lookup = "java:jboss/ee/concurrency/executor/grpc")
```

Comment:
firss ez a branch? ez a masteren már ott van én úgy látom

Reply:
Friss de valamit elcsesztem a rebasenél ahogy látom, localba nem is az én changemnek jelenik meg :))